e2e/src/usecases/WalletConnectV2.js

```diff
@@ -96,10 +96,6 @@ const verifySuccessfulTransaction = async (title = 'Confirm transaction', tx) =>
     .toBeVisible()
     .withTimeout(15 * 1000)
 
-  await expect(element(by.id('WalletConnectRequest/ActionRequestPayload/Value'))).toHaveText(
```

Comment:
i hate to fix a test by removing an assertion, but this part is already covered by unit tests and with the changes in this PR, the tx will use viem to estimate the gas / fee currency. there will no longer be a static tx object to compare against, so i think it makes sense that this assertion is reserved for unit tests rather than e2e